src/javascript/app/base/header.js

```diff
@@ -760,6 +760,13 @@ const Header = (() => {
                 const currentLanguage = Language.get();
                 
                 if (selectedLanguage === currentLanguage) return;
+
+                const allLanguages = Object.keys(Language.getAll());
+                if (!allLanguages.includes(selectedLanguage.toUpperCase())) {
+                    // eslint-disable-next-line no-console
+                    console.warn('Invalid language selected:', selectedLanguage);
```

Comment:
How can an invalid language be selected though?

Reply:
Maybe through url manipulation. We received codeql warning for this, hence to fix that we are adding this to validate and sanitize the selectedLanguage @akmal-deriv